integration_tests/pages/court-register/courtDetails.ts

```diff
@@ -0,0 +1,19 @@
+import page, { Page } from '../page'
+import Chainable = Cypress.Chainable
+
+interface CourtDetailsPage extends Page {
+  markAsClosedButton: () => Chainable<Element>
+  markAsOpenButton: () => Chainable<Element>
+  activatedConfirmationBlock: () => Chainable<Element>
+  deactivatedConfirmationBlock: () => Chainable<Element>
+}
+
+const courtDetails = (courtName: string): CourtDetailsPage =>
+  page(courtName, {
+    markAsClosedButton: () => cy.get('[data-qa=mark-as-closed]'),
+    markAsOpenButton: () => cy.get('[data-qa=mark-as-open]'),
+    activatedConfirmationBlock: () => cy.get('[data-qa=confirm-activation]'),
+    deactivatedConfirmationBlock: () => cy.get('[data-qa=confirm-deactivation]'),
+  }) as CourtDetailsPage
+
```

Comment:
Have to do the `as` `cast` above since the object is dynamic, not sure if there is another way